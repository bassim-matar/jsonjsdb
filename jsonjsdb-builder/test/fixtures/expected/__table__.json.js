jsonjs.data['__table__'] = [['name','lastModif'],['evolution',1757694383],['tag',1738189734],['user',1738189737],['__table__',1757694383]]
