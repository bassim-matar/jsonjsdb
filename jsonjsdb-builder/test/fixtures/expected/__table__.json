[
  {
    "name": "evolution",
    "lastModif": 1757694383
  },
  {
    "name": "tag",
    "lastModif": 1738189734
  },
  {
    "name": "user",
    "lastModif": 1738189737
  },
  {
    "name": "__table__",
    "lastModif": 1757694383
  }
]
